coordinator-common/src/main/java/org/apache/kafka/coordinator/common/runtime/CoordinatorRecord.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.coordinator.group;
+package org.apache.kafka.coordinator.common.runtime;
```

Comment:
Are you going to reuse CoordinatorRecord for the share coordinator? If not, it may be better to keep it in the group-coordinator module for now as it may be specific to the implementation of the coordinator.

Reply:
I will use it